tests/unit/OptionsListUtilsTest.js

```diff
@@ -611,6 +627,44 @@ describe('OptionsListUtils', () => {
         );
     });
 
+    it('getShareDestinationsOptions()', () => {
+        // When we pass an empty search value
+        let results = OptionsListUtils.getShareDestinationOptions(REPORTS, PERSONAL_DETAILS, [], '');
+
+        // Then we should expect 10 recent reports to show because we're grabbing DM chats and group chats
+        expect(results.recentReports.length).toBe(10);
```

Comment:
@cdanwards This PR was merged with this test failing. Can you please have a look at it and fix it?

Reply:
Jack just told me a fix has been merged, so please disregard.